common/src/main/java/com/wynntils/commands/BombCommand.java

```diff
@@ -81,14 +87,15 @@ private int listBombs(CommandContext<CommandSourceStack> context) {
     }
 
     private static MutableComponent getBombListComponent(Set<BombInfo> bombBells) {
-        MutableComponent response = Component.literal("Bomb Bells: ").withStyle(ChatFormatting.GOLD);
+        MutableComponent response = Component.literal("Bombs: ").withStyle(ChatFormatting.GOLD);
 
         if (bombBells.isEmpty()) {
             response.append(Component.literal(
```

Comment:
We should  be able to query the users rank from our Character model... Future improvement, I guess...